Sources/FioriSwiftUICore/Views/TimelineNodesView.swift

```diff
@@ -0,0 +1,60 @@
+import SwiftUI
+
+/// 'TImelineNodesView' displays an image representing a timeline node.
+public struct TimelineNodesView: View {
```

Comment:
```suggestion
public struct TimelineNodeView: View {
```
This view represents one single node. Not multiple nodes. So, it should be singular form, not plural.

Reply:
Updated it.